PowerEditor/src/NppCommands.cpp

```diff
@@ -1377,7 +1377,7 @@ void Notepad_plus::command(int id)
 
 			if (nppGui._fillFindFieldWithSelected)
 			{
-				if (lstrlen(str.get()) <= FINDREPLACE_INSELECTION_THRESHOLD_DEFAULT)
+				if ((nppGui._inSelectionAutocheckThreshold == 0) || (lstrlen(str.get()) < nppGui._inSelectionAutocheckThreshold))
 				{
 					_findReplaceDlg.setSearchText(str.get());

```

Comment:
Use `str.c_str()` instead of `str.get()`.

Reply:
Perhaps the variable name `str` is confusing here...
`str` is not a "wstring", it is a "wchar_t [ ]".
Thus, `.length()` and `.c_str()` are not available to be used.
Please advise on what to do...